buildbot/master.sls

```diff
@@ -58,3 +58,9 @@ buildbot-github-listener:
     - mode: 644
     - watch_in:
       - service: buildbot-github-listener
+
+find /home/servo/buildbot/master/*/*.bz2 -mtime +5 -exec rm {} ;:
```

Comment:
I *think* you can do `-delete` instead of `-exec rm {}` (which may or may not need the semicolon) :P